### PR TITLE
[Chucker Interceptor] Add Chucker Interceptor to snap the API response

### PR DESCRIPTION
**Pull request by lydiasama**
https://github.com/nimblehq/android-templates/projects/3#card-54435439

## What happened 🤔

As PM/Tester, when testing the app, they would like to inspect API request/response to make sure the content sending/receiving to/from the backend in the background correctly.

## Insight 👀

- Add [Chucker](https://github.com/ChuckerTeam/chucker) as an interceptor to intercept the API response

### CODE REVIEW

File: RxJavaTemplate/build.gradle [hunk 1]
```diff
@@ -18,6 +18,8 @@ buildscript {
         android_material_version = '1.2.0-alpha06'
         android_support_version = '1.1.0'
 
+        chucker_interceptor_version = '3.4.0'
+
         constraint_layout_version = '2.0.0-alpha3'
         crashlytics_version = "2.10.1"
```
Comment (luongvo) [inline RxJavaTemplate/build.gradle:24]: they should be in the same group

File: RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/di/modules/RetrofitModule.kt [hunk 2]
```diff
@@ -5,6 +5,7 @@ import co.nimblehq.rxjava.data.service.interceptor.AppRequestInterceptor
 import co.nimblehq.rxjava.data.service.providers.ApiServiceProvider
 import co.nimblehq.rxjava.data.service.providers.ConverterFactoryProvider
 import co.nimblehq.rxjava.data.service.providers.RetrofitProvider
+import com.chuckerteam.chucker.api.ChuckerInterceptor
```
Comment (luongvo) [inline RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/di/modules/RetrofitModule.kt:8]: hmm, redundant import 🤓

**Review by luongvo — APPROVED** [review]
small issues 👏

**Review by luongvo — DISMISSED** [review]

**Comment by lydiasama** [comment]
Oh, Thank you @luongvo  🙏🏻

**Review by chornerman — APPROVED** [review]
lgtm

**Comment by luongvo** [comment]
Test comment

**Review by luongvo — APPROVED** [review]
lgtm

**Review by minhnimble — APPROVED** [review]

**Pull request merged**